x-pack/plugins/reporting/server/export_types/png/execute_job/index.ts

```diff
@@ -37,7 +37,12 @@ export const runTaskFnFactory: RunTaskFnFactory<RunTaskFn<TaskPayloadPNG>> =
             headers,
             urls: [url],
             browserTimezone: job.browserTimezone,
-            layout: job.layout,
+            layout: {
+              ...job.layout,
+              // TODO: We do not do a runtime check for supported layout id types for now. But technically
+              // we should.
+              id: job.layout?.id as PngScreenshotOptions['layout']['id'],
```

Comment:
Just a thought: it seems we could also choose to have this set to `preserve_layout` for all PNG jobs, and just ignore the layout ID from the job params.

Reply:
I like this idea, but currently at a runtime level we default to creating a `print` Layout object if it is unspecified. I'd like to tackle that as a separate PR. Would you mind opening an issue?